src/components/Sidebar/Sidebar.js

```diff
@@ -163,7 +151,9 @@ function Sidebar() {
             <NavLinkStyled
               theme={theme}
               key={button.text}
+              activeClassName="active"
               to={button.link || '/'}
+              exact={true}
```

Comment:
I'm not sure we want exact here, because even when we're on a route like `/characters/:id` we're still going to want to be "within" the characters link and have it active. I think better would be to remove the `Home` link so that we don't have an issue with the incorrect one matching.